README.md

```diff
@@ -1,40 +1,35 @@
 # GitHub code navigation
 
-GitHub code navigation helps you to read, navigate, and understand code by linking definitions of named symbols (like a class or method) to references to that symbol, as well as linking references to the symbol's definition. GitHub has developed two code navigation approaches:
+GitHub code navigation helps you to read, navigate, and understand code by linking definitions of named symbols (like a class or method) to references to that symbol, as well as linking references to the symbol's definition. GitHub code navigation is **search-based:** it uses code search to find all definitions and references across a repository to find symbols with a given name.
```

Comment:
It's a minor nit, but I would probably remove `search-based` here, too. I'm not sure what value that detail adds here.